Modules/Core/SpatialObjects/include/itkEllipseSpatialObject.h

```diff
@@ -82,6 +82,10 @@ class ITK_TEMPLATE_EXPORT EllipseSpatialObject:
   bool IsInsideInObjectSpace(const PointType & point, unsigned int depth=0,
     const std::string & name="" ) const override;
 
+#if ! defined ( ITK_LEGACY_REMOVE )
+    itkLegacyMacro( void SetRadius( ArrayType radii) )
+    { this->SetRadiiInObjectSpace(radii); }
```

Comment:
Just a note that this function is likely to be renamed `SetRadiusInObjectSpace` to correct a failure and be consistent with the current functions in ITK.

Reply:
See #https://github.com/InsightSoftwareConsortium/ITK/pull/640

Reply:
Please read my motivation for "radii" at https://github.com/InsightSoftwareConsortium/ITK/pull/640#issuecomment-479104804

Reply:
Providing backward-compatible interfaces for now.  The larger scope refactoring throughout ITK as discussed by @fbudin69500  seems to have a positive agreement that it will be more linguistically correct and would be welcome as long as it was done completely, uniformly, and with backward compatible legacy migration interface.